src/line_printer.cc

```diff
@@ -97,7 +98,7 @@ void LinePrinter::Print(string to_print, LineType type) {
     };
     vector<CHAR_INFO> char_data(csbi.dwSize.X);
     for (size_t i = 0; i < static_cast<size_t>(csbi.dwSize.X); ++i) {
-      char_data[i].Char.AsciiChar = i < to_print.size() ? to_print[i] : ' ';
+      char_data[i].Char.UnicodeChar = i < w_to_print.size() ? w_to_print[i] : L' ';
```

Comment:
Calling it "UnicodeChar" seems suspicious. Is this from the Windows API?

Reply:
Its from the WinApi

```
typedef struct _CHAR_INFO {
    union {
        WCHAR UnicodeChar;
        CHAR   AsciiChar;
    } Char;
    WORD Attributes;
} CHAR_INFO, *PCHAR_INFO;
```